compiler/src/main/scala/dx/compiler/WorkflowCompiler.scala

```diff
@@ -167,6 +177,168 @@ case class WorkflowCompiler(extras: Option[Extras],
     (commonMeta ++ workflowMeta, commonDetails)
   }
 
+  private def listMd(s: String): String = {
+    "\n* " + s
+  }
+
+  private def listMd2(s: String): String = {
+    "\n    * `" + s + "`"
+  }
+
+  // Summarize reportable dependencies from details of a workflow
+  private def reportableDependenciesFromWorkflow(
+      workflow: DxExecutable
+  ): Option[String] = {
+    val description = workflow.describe(Set(Field.Name, Field.Details, Field.Hidden)) match {
+      case w: DxWorkflowDescribe => w
+      case _                     => throw new Exception(s"Expected ${workflow.id} to be a workflow.")
+    }
+
+    // Do not generate description section if hidden
+    if (description.hidden.isDefined && description.hidden.get) {
+      None
+    }
+
+    // Generate Markdown description section
+    val headerMd = s"\n## Sub-workflow ${description.name}"
+    val details = description.details.collect {
```

Comment:
No need for two separate statements here. Just do `description.details match ...`

Reply:
Somehow, when I try 

```
    val details = description.details match {
      case Some(JsValue(v)) => v.asJsObject match {
        ...
      }
    }
```

I get a compilation error that JsValue can't be found. This is despite the import statement still being 

```
import spray.json._
```

Any idea why that happens?

Reply:
`JsObject`, not `JsValue`

Reply:
Thanks, that works.